# HG changeset patch
# User Rik <rik@octave.org>
# Date 1346288983 25200
#      Wed Aug 29 18:09:43 2012 -0700
# Node ID cef24482e0040dcaaa458599093b8fc527767836
# Parent  e8abaaaa2d7d0763175568130d73fbe394a33ea0
Remove serialization constraint on building pdf and dvi documentation.

* interpreter/Makefile.am: Remove serialization constraint on building pdf and dvi documentation.

diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -160,21 +160,16 @@ octave_TEXINFOS = \
   $(EXAMPLE_FILES) \
   $(IMAGES) \
   $(LOGOS) \
   $(srcdir)/images.mk
 
 octave.html:
 	if [ -d octave.html ]; then true; else mkdir octave.html; fi
 
-## The texi2dvi script (used to create both PDF and DVI output formats)
-## uses some fixed temporary file names, so wait for DVI file to be
-## generated before creating PDF file to avoid conflicts.
-octave.pdf: octave.dvi
-
 all-local: dvi html pdf ps doc-cache $(HTMLDIR_IMAGES)
 
 # Prevent packaging of distribution unless all libraries 
 # necessary to create documentation are present
 dist-hook:
 	@$(GREP) '#define HAVE_COLAMD 1' $(top_builddir)/config.h > /dev/null || { echo "Documentation creation requires missing COLAMD library.  Cannot package distribution!" ; exit 1; }
 	@$(GREP) '#define HAVE_CHOLMOD 1' $(top_builddir)/config.h > /dev/null || { echo "Documentation creation requires missing CHOLMOD library.  Cannot package distribution!" ; exit 1; }
 	@$(GREP) '#define HAVE_UMFPACK 1' $(top_builddir)/config.h > /dev/null || { echo "Documentation creation requires missing UMFPACK library.  Cannot package distribution!" ; exit 1; }
